internal/app/machined/pkg/runtime/v1alpha1/bootloader/meta.go

```diff
@@ -117,6 +117,10 @@ func (m *Meta) Revert() (err error) {
 		return err
 	}
 
+	if conf == nil {
```

Comment:
seems, this was missed in this file, all other places checked if the conf was nil :+1: